_content/articles/gleanings-from-the-mahavastu_marciniak-katarzyna.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Gleanings from the Mahāvastu"
+authors:
+  - "Katarzyna Marciniak"
+external_url: "https://www.academia.edu/59240112/Gleanings_from_the_Mah%C4%81vastu"
```

Comment:
https://archive.org/download/gleanings-from-the-mahavastu-marciniak/Gleanings%20from%20the%20Mahavastu%20-%20Marciniak.pdf